src/util/index.js

```diff
@@ -129,17 +129,28 @@ export const descriptionPlaceholder = length => {
  */
 export const hexToRgba = ( hexColor, opacity = null ) => {
```

Comment:
These code should not be placed here. The `hexToRgba` function should only do one thing, convert hex to rgba. It shouldn't perform any global color functionality. Keep the global color functionality in the global color area